recipes/csm/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "libcsm" %}
+{% set version = "3.0.3" %}
+{% set commit = "9b11afac782fe210003310c9868af8480ab05c9c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/csm/archive/{{ commit }}.tar.gz
```

Comment:
This appears to be a fork and using a commit. Can you explain why?

Reply:
We're using the fork and a specific commit because it has a cmake build system for linux and mac in it. We're working with the group that manages the Communiy Sensor Model to get the cmake build system integrated into their repository.

Reply:
Yeah that sounds like the proper way to go about this.